news/2016-11-02-recognising-the-best-of-the-best.md

```diff
@@ -13,71 +13,93 @@ The osu! community is comprised of hundreds of thousands of people. Among this h
 
 Community contributors are a select pick of long-term members of the community that have given huge portions of their time and effort towards making things more awesome for people. While not every effort they make is visible to all people, recipients of the badge are undoubtedly a positive influence on the community and exemplify everything that we could ever want from the osu! community as a whole.
 
-<img src="https://a.ppy.sh/3178418" style="width: 120px; height: 120px">
-<p style="text-align:center;"><b><a href="https://osu.ppy.sh/users/3178418">pishifat</a></b></p>
-<p style="text-align:center;"><i>Exemplary performance in beatmap construction and education</i></p>
+<div align="center">
+    <img src="https://a.ppy.sh/3178418" style="width: 120px; height: 120px">
```

Comment:
You might want to wrap the images in `<p></p>` tags, spacing will be broken otherwise if my web-fu isn't incorrect:

![image](https://user-images.githubusercontent.com/20418176/74104058-66f52f00-4b51-11ea-8ec7-d02beba4b3a2.png)

Reply:
applied